src/Websites/Websites/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 * Added support for new access restriction features: ServiceTag, multi-ip and http-headers
```

Comment:
Did the Access-resriction change not already release?

Reply:
Not Sure sisira. i have taken the latest and it is under Upcoming Release.

Reply:
Modified as per release plan.